packages/react/slider/src/Slider.tsx

```diff
@@ -678,40 +679,6 @@ const BubbleInput = (props: React.ComponentProps<'input'>) => {
   return <input style={{ display: 'none' }} {...inputProps} ref={ref} />;
 };
 
-function useDirection({
```

Comment:
I moved this into its own package too as it seems like something we should implement in all of our comps that support `rtl` when we find time.